c_src/sqlite3_nif.c

```diff
@@ -700,7 +700,14 @@ exqlite_insert_all(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
     }
 
     int stmt_param_count = sqlite3_bind_parameter_count(statement->statement);
-    int types_array[stmt_param_count];
+
+    // TODO: MSVC can't do VLA
+    // int types_array[stmt_param_count];
+    int* types_array = (int*)malloc(stmt_param_count * sizeof(int));
```

Comment:
Use `exqlite_malloc` for this. This way the VM will know about the memory allocated and will count it correctly. Also `exqlite_free` to free the memory.

Reply:
Right, I did it in the very next commit: https://github.com/elixir-sqlite/exqlite/pull/306/commits/cf69cd54cec3d111b7e648deac36f028d04ad227

Reply:
ah I was walking the changes.